packages/renderer-react/src/browser.tsx

```diff
@@ -161,7 +161,7 @@ export function renderClient(opts: {
             }
           }
           // server loader
-          if (!isFirst && opts.routes[id].hasServerLoader) {
+          if (!isFirst && opts.routes[id]?.hasServerLoader) {
```

Comment:
```suggestion
          // use ?. since routes patched with patchClientRoutes is not exists in opts.routes
          if (!isFirst && opts.routes[id]?.hasServerLoader) {
```